packages/admin/cms-admin/src/common/MasterMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import { Menu, MenuCollapsibleItem, MenuContext, MenuItemRouterLink, MenuItemRouterLinkProps, useWindowSize } from "@comet/admin";
+import * as React from "react";
+import { RouteProps, useRouteMatch } from "react-router-dom";
+
+export type MasterMenuItem = Omit<MenuItemRouterLinkProps, "to"> & {
+    route?: RouteProps;
+    to?: string;
+    subMenu?: MasterMenuItem[];
```

Comment:
MUI typically use sub-props as a single word, for instance, `subheader`. Should we do so as well? Then it would be `submenu`. I think both is correct in English though.

Reply:
645a0a259a4095456b49f57e7d3b16511ec97ffd